pkg/client/result.go

```diff
@@ -14,6 +16,31 @@ type Result struct {
 	Rows    []Row    `json:"rows"`
 }
 
+// Due to big int number limitations in javascript, numbers should be encoded
+// as strings so they could be properly loaded on the frontend.
+func (res *Result) PrepareBigints() {
+	for i, row := range res.Rows {
+		for j, col := range row {
+			if col == nil {
+				continue
+			}
+
+			switch reflect.TypeOf(col).Kind() {
+			case reflect.Int64:
+				val := col.(int64)
+				if val < -9007199254740991 || val > 9007199254740991 {
+					res.Rows[i][j] = strconv.FormatInt(col.(int64), 10)
+				}
+			case reflect.Float64:
+				val := col.(float64)
+				if val < -999999999999999 || val > 999999999999999 {
+					res.Rows[i][j] = strconv.FormatFloat(val, 'e', -1, 64)
+				}
```

Comment:
[All numbers](http://stackoverflow.com/questions/3605925/does-javascript-have-double-floating-point-number-precision) in Javascript are by double precision floating point, this conversion shouldn't be necessary at all, and will probably cause similar bugs to my comment above.